gallery/src/components/NxStatefulTreeViewMultiSelect/NxStatefulTreeViewMultiSelectPage.tsx

```diff
@@ -73,9 +73,11 @@ const NxStatefulTreeViewMultiSelectPage = () =>
             <td className="nx-cell">Yes</td>
             <td className="nx-cell">
               Called whenever selection change occurs; it will receive two arguments:
-              <ul>
-                <li><code className="nx-code" >Set</code> of ids of the currently selected options</li>
-                <li>
+              <ul className="nx-list nx-list--bulleted">
+                <li className="nx-list__item">
+                  <code className="nx-code">Set</code> of ids of the currently selected options
+                </li>
+                <li className="nx-list__item"> className="nx-list__item"
```

Comment:
Some weird copy/paste thing going on here? 
```suggestion
                <li className="nx-list__item">
```